cerebrium/fine-tuning/diffusion-models/dataset.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: "Dataset Curation"
+description: "Guide to curating a dataset for success when fine-tuning your diffusion model on Cerebrium."
+---
+
+<Note>
+  Cerebrium's fine-tuning functionality is in public beta and we are adding
+  more functionality each week! If there are any issues or if you have an urgent requirement, please reach
+  out to [support](mailto:support@cerebrium.ai) 
+</Note>
+
+Curating your dataset is arguably the most important step in fine-tuning a diffusion model.  
+The quality of your dataset will determine the quality of your outputs, so spending extra time on this step is highly recommended.
+
+## Curation guidelines
+
+The following is a set of guidelines to keep in mind when curating your dataset:
+- Images should be sized to 512 x 512 for stable diffusion v1.5. 
+  - If not, we'll resize them for you, but this may result in a loss of quality.
+- Your images should all be upright or in the same orientation.
+- Keep the number of training images low. Aim for between 10 and 15 as your model may not converge if you use too many.
+- Make sure your images are of the same class. For example, if you are fine-tuning the prompt "Photo of a dog", all of your images should be of dogs.
+
+
+
+If you are doing style-based fine-tuning:
+- Try fine-tuning a single style at a time. Keep the style consistent across all of your images.
```

Comment:
so minor but could be 3 points:

- Fine-tuning a single style at a time
- Keep the style consistent across all of your images
- Keep the colouring consistent across all of your images